### PR TITLE
add auto-generated Color page to Gatsby site!

### PR DESCRIPTION
**Pull request by afebbraro**
## What does this PR do?
Add Color page to Gatsby site and two new shades of red.

There will be a follow-up task to finish the A11y and Dos/donts section.

### Associated Issue 
#2372 and #2419 

### Code
 - [x] Build Component in Spark Gatsby Site

### Accessibility
- [ ] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)
---- Will be working on this with olya once back from ooo

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [x] Apple Safari (Mobile)

### CODE REVIEW

**Comment by afebbraro** [comment]
This is ready for review but will have additional work done for the Do's / Don't and a11y sections. Everything but those sections can be reviewed.

File: src/components/ColorSwatch.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      sprk-o-Stack--large
+      "
+    >
+      <h3 className="sprk-b-TypeDisplayFive sprk-o-Stack__item">
+        {name}
+      </h3>
+
+      <p className="sprk-b-TypeBodyTwo sprk-o-Stack__item">
+        {description}
+      </p>
```
Comment (bobwise) [inline src/components/ColorSwatch.js]: I don't think this is providing any info that we're not already getting from the name. I think we can delete this.

Comment (bobwise) [inline src/components/ColorSwatch.js]: I don't think this is providing any value that isn't already in name. I think we can delete this.

File: src/components/Header.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const Header = ({ menuVisible, setMenuVisible, context, setContext }) => (
   <header className="docs-header">
     <div className="docs-header__group">
       <button className="docs-header__menu-button" onClick={() => { setMenuVisible(!menuVisible); }}>
+        <span className="sprk-u-ScreenReaderText">Toggle Navigation</span>
```
Comment (bobwise) [inline src/components/Header.js]: 👍

File: src/components/ColorSwatch.js (hunk 1 above)
Comment (afebbraro) [reply src/components/ColorSwatch.js]: hey @bobwise thanks for the comment, the description is something that I need to nail down with Jane. I think we might use this for something like Usage. Which could be really helpful to show someone when to use a particular color. If after syncing with Jane, we decide we don't need it then I'll remove it.

**Comment by afebbraro** [comment]
@bobwise  you mentioned maybe separating out the package version updates. I think I should include them here cause they're needed to be able to view this color page without having to wait 10 mins. I tested out the gatsby site and didn't see any issues with the new update, but if something comes we can always roll back, update again or fix it in a follow up PR.
Since we are in rapid development with this stuff, I've been updating Storybook and other deps as much as I possibly can in hopes to keep us up to date. Once we have a stable build that's ready for prod (when we get all our component content into gatbsy) then I think we should absolutely be more mindful about doing version updates. (We talked IRL about this!) Woot!!

**Comment by yodasw16** [comment]
@afebbraro I missed the time to submit a review, but I think it needs more space between the paragraph at the top and the images. I would do `mbh`. On small view ports the white background of the image creates extra space that isn't there when the images are side-by-side. Maybe put a light grey border around the white image or a box shadow like the cards have so that you can see the edges?

**Pull request merged**